addons/web/static/src/js/views/kanban/kanban_quick_create.js

```diff
@@ -117,16 +117,20 @@ var RecordQuickCreate = AbstractQuickCreate.extend({
     /**
      * @override
      */
-    init: function (parent, width) {
+    init: function (parent, width, options) {
```

Comment:
you should put the 'width' argument into the options: init: function (parent, options)